app/main.ipynb

```diff
@@ -130,7 +136,8 @@
    "outputs": [],
    "source": [
     "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1])\n",
-    "# write your code here"
+    "sum_subseq = array[[0, 1, 2, 3, 4]].sum()\n",
+    "print(\"sum of subsequence: \", sum_subseq)"
```

Comment:
The sum of the subsequence does not match the expected result of 27. You need to select the correct indices from the array to achieve the desired sum.